### DISCUSSION TITLE
Where is the documentation?

### DISCUSSION THREAD

**Discussion by bhakops-console00** [react-native-image-picker/react-native-image-picker#1618] [Q&A]
Hi everyone.
I couldn't find the documentation for 3.x.x.
Where can I find it?
![image-picker](https://user-images.githubusercontent.com/73786621/107886176-4bb0c280-6f0f-11eb-9917-694f152ddeda.PNG)

**Comment by Johan-dutoit** [comment]
vnext lives on the main branch.
